drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/ActionWorkItemPlugin.java

```diff
@@ -173,10 +173,25 @@ public PortableWorkDefinition getWorkItemDefinition() {
     public void forEachWorkItem(final BiConsumer<String, String> biConsumer) {
         getPresenter()
                 .getWorkItemDefinitions()
+                .stream()
+                .filter(this::isWorkItemAvailable)
                 .forEach(workDefinition -> biConsumer.accept(workDefinition.getDisplayName(),
                                                              workDefinition.getName()));
     }
 
+    private boolean isWorkItemAvailable(final PortableWorkDefinition pwd) {
+        if (isNewColumn()) {
```

Comment:
I don't agree with this `if` and the `testForEachWorkItemWhenOneHasBeenExecutedUpdatedColumn` test. Thanks to this you can still invoke two same WI. 

Follow these steps: 
- Define two WI (WorkItemOne, WorkItemTwo) definitions
- In the same package create GDT
- Add Execute WI column - select WorkItemOne
- Add Execute WI column - select WorkItemTwo
- Edit the first added column - you can select WorkItemTwo
- Now you are in state when two same WI are executed